cmd/asset/list/list.go

```diff
@@ -66,5 +78,58 @@ func NewCmdList() *cobra.Command {
 		},
 	}
 
+	cmd.Flags().StringSliceVar(&opt.input.OpponentMultisig.Receivers, "receivers", nil, "multisig receivers")
+	cmd.Flags().Uint8Var(&opt.input.OpponentMultisig.Threshold, "threshold", 0, "multisig threshold")
+
 	return cmd
 }
+
+func readMultisignAssets(ctx context.Context, client *mixin.Client, input mixin.TransferInput) ([]*mixin.Asset, error) {
+	assets, err := mixin.ReadMultisigAssets(ctx)
+	if err != nil {
+		return nil, err
+	}
+	assetMap := map[string]*mixin.Asset{}
+	for _, asset := range assets {
+		assetMap[asset.AssetID] = asset
+	}
+
+	result := []*mixin.Asset{}
+	resultMap := map[string]*mixin.Asset{}
+	offset := time.Time{}
+	for {
+		outputs, err := client.ListMultisigOutputs(ctx, mixin.ListMultisigOutputsOption{
+			Members:        input.OpponentMultisig.Receivers,
+			Threshold:      input.OpponentMultisig.Threshold,
+			Offset:         offset,
+			Limit:          500,
+			OrderByCreated: true,
+			State:          mixin.UTXOStateUnspent,
+		})
+		if err != nil {
+			return nil, err
+		}
+		if len(outputs) == 0 || offset.Equal(outputs[len(outputs)-1].CreatedAt) {
+			break
+		}
+		if outputs[0].CreatedAt.Equal(offset) {
+			outputs = outputs[1:]
+		}
+
+		for _, output := range outputs {
+			a := resultMap[output.AssetID]
+			if a == nil {
+				a = assetMap[output.AssetID]
+				if a == nil {
+					continue
```

Comment:
Not sure if this is to ignore or add a new mixin.Asset, such as Panda Seed, it's not supported by multisign, is it necessary to display it? (or add a new flag to decide display or not)

Reply:
Yes, it is necessary.  ERC20 tokens are all supported by mixin mainnet (multisign).

Reply:
Updated.